src/tools/msc.lua

```diff
@@ -398,3 +407,13 @@
 
 		return result
 	end
+
+
+	function msc.getexternalincludedirs(cfg)
```

Comment:
To be coherent with other toolset, that implementation should be included in `getincludedirs`
and just call `msc.getincludedirs(cfg, nil, nil, cfg.externalincludedirs, nil)`

Reply:
Will fix.